src/riak_kv_replrtq_snk.erl

```diff
@@ -530,18 +529,7 @@ map_peer_to_wi_fun({QueueName, Iteration, PeerInfo}) ->
                                 {certfile, CertfiicateFilename},
```

Comment:
This variable name has a typo, but nothing you did add.

Reply:
Will still change though